includes/functions.php

```diff
@@ -966,15 +966,17 @@ function ap_question_title_with_solved_prefix( $question_id = false ) {
 	return get_the_title( $question_id );
 }
 
-/**
- * Verify the __nonce field.
- *
- * @param string $action Action.
- * @return bool
- * @since  2.4
- */
-function ap_verify_nonce( $action ) {
-	return wp_verify_nonce( ap_sanitize_unslash( '__nonce', 'p' ), $action );
+if ( ! function_exists( 'ap_verify_nonce' ) ) {
```

Comment:
@mebishalnapit Unfortunately, integrating other plugin functions here isn't feasible. It poses a risk of disrupting our ajax actions. To address this, we'll need to deprecate the current function and introduce a new one, anspress_verify_nonce. Then, we'll have to systematically replace all instances of calls to the ap_verify_nonce function throughout our plugin.

Reply:
@rahularyan Done.